src/oaklib/converters/obo_graph_to_fhir_converter.py

```diff
@@ -80,19 +80,22 @@ def dump(
         self,
         source: GraphDocument,
         target: str = None,
+        output_as_npm: bool = False,
```

Comment:
@cmungall This throws a monkey wrench into moving params down to `convert()`. I think that `output_as_npm` is best placed in `dump()`.

Reply:
alternatively, consider reserving `fhirjson` for an actual JSON document, and create a new output type `fhirnpm`

Reply:
Hmm, that is a good suggestion. Let's go with that!

Reply:
Going to mark this one 'resolved' as I've implemented `fhirnpm` and a new `OboGraphToFhirNpmConverter` class. I'll add review comments to `OboGraphToFhirNpmConverter`.